### ISSUE TITLE
Update `pkg` port

### ISSUE THREAD

**Issue by 0xThiebaut** [opnsense/ports#207]
On a stock `14.1-RELEASE`, `make ports` fails due to a mismatch of the installed `pkg` version.

```
>>> Running build step: ports
>>> Passing arguments: (none)
Installed pkg version '1.21' does not match required version '1.19'
*** Error code 1
```

``` 
# pkg --version
1.21.3
# freebsd-version
14.1-RELEASE
```

On a side-note, where could I find the documentation to bump the package myself?

**Comment by fichtner** [comment]
Yes, you need to reinstall pkg from our ports tree to ensure a compatible build.

**Comment by 0xThiebaut** [comment]
Thanks for the info, seems I missed that part. I would recommend adding this to the [opnsense/tools](https://github.com/opnsense/tools?tab=readme-ov-file#setting-up-a-build-system) documentation which just mentions "FreeBSD 14.1-RELEASE for amd64" as a build dependency.
